source/Halibut.Tests/Support/SerilogLoggerBuilder.cs

```diff
@@ -75,7 +75,12 @@ public static string CurrentTestHash()
         {
             using (SHA256 mySHA256 = SHA256.Create())
             {
-                return Convert.ToBase64String(mySHA256.ComputeHash(TestContext.CurrentContext.Test.FullName.GetUTF8Bytes()))
+                string s = TestContext.CurrentContext.Test.FullName;
```

Comment:
Is this running locally, or on TeamCity?

I'm guessing it won't cause any issues running them in parallel?